torch/linalg/__init__.py

```diff
@@ -214,6 +214,86 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+cond = _add_docstr(_linalg.linalg_cond, r"""
+linalg.norm(input, p=None, *, out=None) -> Tensor
```

Comment:
In native_functions.yaml the signature has `ord` instead of `p` -- I think it just needs to be updated for consistency?

Reply:
It also would be much better if `linalg.cond` and `linalg.norm` had the same argument name for describing the norm type, and `ord` and `p` aren't good names, since for example Frobenius norm is neither an order `ord` nor the power `p` of the norm.